modules/drawXarm.py

```diff
@@ -286,7 +286,7 @@ def set_speed(self, arm_speed):
     All of the notation functions ()below) need to call these here.
     Only 4 are required:
         arc: make an arc (inc circle)
-        move_to: draw a line of move to in a linear way to another coord
+        bot_move_to: draw a line of move to in a linear way to another coord
         jump_to: will arc between coords lifting z an amount (arc radius)
```

Comment:
jump_to doesnt actually do this on the z axis.

Reply:
I've actually set the z height for the jump (as opposed to have it as an argument) with xArm so that it matches the Dobot code